citrination_client/base/base_client.py

```diff
@@ -28,10 +28,18 @@ def __init__(self, api_key, webserver_host, api_members=[], suppress_warnings=Fa
         :param suppress_warnings: A flag indicating whether or not warning
             messages to stdout should be printed
         :type suppress_warnings: bool
+        :param proxies: proxies to use when making HTTP requests. E.g.,
+                proxies = {
+                  'http': 'http://10.10.1.10:3128',
```

Comment:
Where do these IP:PORT numbers come from?

Reply:
Hah good catch. They come from the requests API doc (http://docs.python-requests.org/en/latest/user/advanced/#proxies)